src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.EntityMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.OverseasEntityDueDiligenceMock;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.PresenterMock;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.EntityDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class OverseasEntitySubmissionDtoValidatorTest {
+
+    private static final String CONTEXT = "12345";
+    @InjectMocks
+    private OverseasEntitySubmissionDtoValidator overseasEntitySubmissionDtoValidator;
+    @Mock
+    private AddressDtoValidator addressDtoValidator;
+    @Mock
+    private EntityDtoValidator entityDtoValidator;
+    @Mock
+    private PresenterDtoValidator presenterDtoValidator;
+    @Mock
+    private OverseasEntityDueDiligenceValidator overseasEntityDueDiligenceValidator;
+    @Mock
+    private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+
+    @Test
+    void testOverseasEntityDueDiligenceValidator() {
+        buildOverseasEntitySubmissionDto();
+        overseasEntitySubmissionDtoValidator.validate(overseasEntitySubmissionDto, new Errors(), CONTEXT);
+        verify(entityDtoValidator, times(1)).validate(any(),any(),any());
```

Comment:
you should be able to verify the first argument if you assign EntityMock.getEntityDto() to a variable or class level variable (same applies for presenter and due diligence validators)

Reply:
you'll have to use eq(....) if using any() matchers for the other params

Reply:
Updated the test